src/components/controls/buttons/LinkButton/LinkButton.stories.tsx

```diff
@@ -8,28 +8,31 @@ import { LinkButton } from "./LinkButton"
 export default {
   title: "Controls/Buttons/LinkButton",
   component: LinkButton,
-} as ComponentMeta<typeof LinkButton>
+} satisfies ComponentMeta<typeof LinkButton>
 
-const Template: ComponentStory<typeof LinkButton> = (args) => <LinkButton {...args} />
+type Story = ComponentStoryObj<typeof LinkButton>
 
-export const Basic = Template.bind({})
-Basic.args = {
-  children: "Button",
-}
-Basic.play = async ({ canvasElement }) => {
-  const canvas = within(canvasElement)
-  const button = canvas.getByRole("button", { name: "Button" })
-  await expect(button).not.toBeDisabled()
-}
+export const Basic = {
+  args: {
+    children: "Button",
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement)
+    const button = canvas.getByRole("button", { name: "Button" })
+    await expect(button).not.toBeDisabled()
+  },
+} satisfies Story
 
-export const Snapshot = (): JSX.Element => (
-  <SnapshotWrapper>
-    {[Basic].map((Story, index) => (
-      // eslint-disable-next-line react/no-array-index-key
-      <Story key={index} {...(Story.args as ComponentProps<typeof LinkButton>)} />
-    ))}
-  </SnapshotWrapper>
-)
-Snapshot.parameters = {
-  chromatic: { disableSnapshot: false },
-}
+export const Snapshot = {
+  render: () => (
+    <SnapshotWrapper>
+      {[Basic].map((Story, index) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <LinkButton key={index} {...(Story.args as ComponentProps<typeof LinkButton>)} />
```

Comment:
I found that type assertion is no longer necessary with CSF3:

```suggestion
        <LinkButton key={index} {...Story.args} />
```

(Applies to all stories in PR)

Reply:
For `Calendar` and `RangeCalendar` I still need it 🤔